corehq/apps/hqwebapp/static/hqwebapp/less/_hq/select2s.less

```diff
@@ -57,13 +57,25 @@
 .select2-container {
     width: 100% !important;
 
+    .select2-selection.select2-selection--single {
+        height: @input-height-base;
+    }
+
+    .select2-selection.select2-selection--multiple {
+        min-height: @input-height-base;
+    }
+
     .select2-search-field {
         width: 100% !important;
     }
 
     .select2-input {
         width: 100% !important;
     }
+
+    textarea.select2-search__field {    // Placeholder for multi-select
+        line-height: 21px;
```

Comment:
Where does 21px come from (or what does it correspond to)?

Reply:
It'd be good for it not to be hard-coded, but it doesn't really relate to anything else. It's just from me futzing with styles by hand to center the text vertically. There are alternatives - change the height of the textarea, or its margin - but they would be similar ad hoc values.